### PR TITLE
Adding layout redesign branch to repository

### PR DESCRIPTION
**Pull request by tommylcameron**
1. Changed site background to wine colored gradient
2. Installed font awesome gem
3. More modern, sans-serif, white font

### CODE REVIEW

File: app/assets/javascripts/index.js [hunk 1]
```diff
@@ -6,10 +6,8 @@ function bindFilterToColumn(column_slug) {
         var trows = document.getElementsByClassName("trow");
         Array.prototype.forEach.call(trows, function(row) {
             var column = row.getElementsByClassName(column_class)[0];
-
             var searchString = filter.value.toLowerCase();
             var columnValue = column.textContent.toLowerCase();
-
```
Comment (arvinds) [inline app/assets/javascripts/index.js]: Were these changes intended?

File: app/assets/stylesheets/application.css.scss [hunk 2]
```diff
@@ -13,8 +13,19 @@
...
  */
+@import "font-awesome-sprockets";
+@import "font-awesome";
+ 
 body {
-    background-image: url(http://www.weightlossreviewshub.com/wp-content/uploads/2016/03/Red-Wine.jpg);
-    font-family: "Cormorant Garamond", serif;
+    background: #C04848; /* fallback for old browsers */
+    background: -webkit-linear-gradient(to left, #C04848 , #480048); /* Chrome 10-25, Safari 5.1-6 */
+    background: linear-gradient(to left, #C04848 , #480048); /* W3C, IE 10+/ Edge, Firefox 16+, Chrome 26+, Opera 12+, Safari 7+ */
```
Comment (arvinds) [inline app/assets/stylesheets/application.css.scss]: Did you consider other gradients like radial gradients?

File: app/views/beverages/_form.html.erb [hunk 3]
```diff
@@ -1,37 +1,39 @@
...
-      <%= b.text_field :reporter %>
-    </p>
-    <p>
-      <%= b.submit @action.capitalize %><br>
-    </p>  
-<% end %>
\ No newline at end of file
+<div>
+  <%= form_for @beverage, url: {action: @action} do |b| %>
+      <p  class="whitetext">
```
Comment (arvinds) [inline app/views/beverages/_form.html.erb]: You could just apply the whitetext css to all `<p>` tags, instead of adding the class to all paragraph tags

File: app/assets/javascripts/index.js (hunk 1 above)
Comment (tommylcameron) [inline app/assets/javascripts/index.js]: They weren't intended. The spaces have been removed

**Comment by tommylcameron** [comment]
@arvinds I've made the changes per your feedback. Please review.

**Comment by arvinds** [comment]
@tommylcameron looks good!
